react-app/src/pages/AccessRequest.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Box, Button, Grid, Paper, Typography } from '@mui/material';
+import TextInput from '@/components/form/TextFormField';
+import AutocompleteFormField from '@/components/form/AutocompleteFormField';
+import { useKeycloak } from '@bcgov/citz-imb-kc-react';
+import { FormProvider, useForm } from 'react-hook-form';
+
+export const AccessRequest = () => {
+  const keycloak = useKeycloak();
+  const formMethods = useForm({
+    defaultValues: {
+      UserName: keycloak.state.userInfo.idir_username,
```

Comment:
A note for later. BCeID users won't have this, so we'll have to do a check beforehand.